### ISSUE TITLE
PHP 5.6 operator ** leads to parse error, PHP-Parser dependency update needed

### ISSUE THREAD

**Issue by fi5er** [phpDocumentor/phpDocumentor2#1727]
PHP 5.6 has added the exponentiation operator `**`, but the version of [PHP-Parser](https://github.com/nikic/PHP-Parser) version used by phpDocumentor2 (v2.5.8) doesn't support it. The current PHP-Parser version does support it, as well as `**=` (and PHP 7, too). PHP 5.6 was released in August 2014.

```
Collecting files .. OK
Initializing parser .. OK
Parsing files
Parsing /path/to/testfile.php
PHP Notice:  Undefined offset: 300 in /usr/share/php/phpDocumentor/vendor/nikic/php-parser/lib/PHPParser/Lexer.php on line 121

Notice: Undefined offset: 300 in /usr/share/php/phpDocumentor/vendor/nikic/php-parser/lib/PHPParser/Lexer.php on line 121
PHP Notice:  Undefined index:  in /usr/share/php/phpDocumentor/vendor/nikic/php-parser/lib/PHPParser/Parser.php on line 983

Notice: Undefined index:  in /usr/share/php/phpDocumentor/vendor/nikic/php-parser/lib/PHPParser/Parser.php on line 983
PHP Notice:  Undefined index:  in /usr/share/php/phpDocumentor/vendor/nikic/php-parser/lib/PHPParser/Parser.php on line 1099

Notice: Undefined index:  in /usr/share/php/phpDocumentor/vendor/nikic/php-parser/lib/PHPParser/Parser.php on line 1099
Parse Error: Syntax error, unexpected  on line 53  No summary was found for this file
Storing cache in "/path/to/output" .. OK
Load cache                                                         ..    0.000s
[... rest runs ok ...]
```

**Comment by jaapio** [comment]
Upgrading the parser would require more testing than our test-suite provides. All our effort is going to phpdocumentor 3 which is currently in development. We have limited resources so we won't be able to fix this soon. I'm sorry about that.

**Comment by fi5er** [comment]
I see. Is there a rough ETA on v3?

**Comment by jaapio** [comment]
releated to #1701. issue will be solved when php 5.6+ is supported
